src/test/java/com/cdancy/bitbucket/rest/features/RepositoryApiLiveTest.java

```diff
@@ -177,6 +180,79 @@ public void testDeletePermissionByGroup() {
         assertThat(success).isTrue();
     }
 
+    @Test(dependsOnMethods = {"testGetRepository", "testCreateRepository"})
+    public void testListHook() {
+        HookPage hookPage = api().listHooks(projectKey, repoKey, 0, 100);
+        assertThat(hookPage).isNotNull();
+        assertThat(hookPage.errors()).isEmpty();
+        assertThat(hookPage.size()).isGreaterThan(0);
+        for (Hook hook : hookPage.values()) {
+            if (hook.details().configFormKey() == null) {
```

Comment:
If configFormKey is not null, test need to configure it before enableHook. For configurate hook,  I need to implements each rest call for all hooks.